assets/stylesheets/objects/_grid.scss

```diff
@@ -32,6 +38,15 @@ $gutter-width-fluid: 2%;
     }
 }
 
+.grid__item--flex-2-columns {
+    border: 0 !important;
```

Comment:
Not very happy with this use of `!important`. The lack of a border has nothing to do with the fact that this is 2-column flex - it's entirely specific to your particular use case. I would add a utility class like `.u-no-border` to override border styles and add this class to the element in question